### PR TITLE
Bump sane-pdf-reports docker image

### PR DESCRIPTION
**Pull request by lmichalevich**
Related PRs:
- https://github.com/demisto/dockerfiles/pull/9363

### CODE REVIEW

**Review by davidbinyamin — APPROVED** [review]
dont forget doc approve

File: Packs/Base/ReleaseNotes/1_31_29.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#### Scripts
+##### SanePdfReports
+- Updated the Docker image to: *demisto/sane-pdf-reports:1.0.0.36824*.
+  - Fix bug that panels were overlapping on each other
```
Comment (amshamah419) [inline Packs/Base/ReleaseNotes/1_31_29.md:4]:
```suggestion
  - Fixed an issue where panels would overlap each other.
```

**Review by amshamah419 — APPROVED** [review]

**Comment by lmichalevich** [comment]
Closing it because there is another sane-reports bug that I want to fix

File: Packs/Base/ReleaseNotes/1_31_29.md (hunk 1 above)
Comment (lmichalevich) [reply Packs/Base/ReleaseNotes/1_31_29.md:4]: @amshamah419 thanks.
I also want to add 'Fixed an issue where the report date was missing'
is it ok?

Comment (amshamah419) [reply Packs/Base/ReleaseNotes/1_31_29.md:4]: That sounds alright.

**Pull request closed without merging**